ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -133,9 +135,12 @@ export const WorkspaceAbout = fp.flow(withUserProfile(), withUrlParams())
 
   async loadUserRoles() {
     const {workspace} = this.state;
+    this.setState({loadingWorkspaceUserRoles: true});
     workspacesApi().getFirecloudWorkspaceUserRoles(workspace.namespace, workspace.id).then(
       resp => {
-        this.setState({workspaceUserRoles: fp.sortBy('familyName', resp.items)});
+        this.setState({
+          workspaceUserRoles: fp.sortBy('familyName', resp.items),
+          loadingWorkspaceUserRoles: false});
       }
     ).catch(error => {
       console.error(error);
```

Comment:
See my comment below about possibly removing the `loadingWorkspaceUserRoles` variable, but if you decide to keep it, do you think we should set it to `false` here in the event of an error?

Reply:
I don't think so, because if something went wrong fetching the roles i think we still want to display a spinner (rather than out-of-date roles or no roles).  Something to implement in the future is a better UX for that failure case (we have a bad habit of quietly swallowing a lot of our api call failures, and a seemingly-arbitrary subset where we show a popup or error banner.  If there is not already a ticket for auditing that I will make one).